l10n_it_declaration_of_intent/views/declaration_of_intent_view.xml

```diff
@@ -0,0 +1,197 @@
+                            />
+                            <field
+                                name="partner_document_number"
+                                attrs="{'readonly':[('state','!=','valid')]}"
+                            />
+                            <field
+                                name="partner_document_date"
+                                attrs="{'readonly':[('state','!=','valid')]}"
+                            />
+                        </group>
+                        <group name="valid_period" string="Valid Period">
+                            <field
+                                name="date_start"
+                                attrs="{'readonly':[('state','!=','valid')]}"
+                            />
+                            <field name="date_end" />
+                        </group>
+                    </group>
+
+                    <group name="accounting" string="Accounting">
+                        <field
+                            name="fiscal_position_id"
+                            widget="selection"
+                            attrs="{'readonly':[('state','!=','valid')]}"
+                        />
+                        <field
+                            name="taxes_ids"
+                            widget="many2many_tags"
+                            attrs="{'readonly':[('state','!=','valid')]}"
+                        />
+                    </group>
+
+                    <div>
+                        <label for="limit_amount" class="oe_inline" />
+                        <field name="used_amount" class="oe_inline" nolabel="1" />
+                        /
+                        <field
+                            name="limit_amount"
+                            nolabel="1"
+                            class="oe_inline"
+                            attrs="{'readonly':[('state','!=','valid')]}"
+                        />
+                        <field
+                            name="currency_id"
+                            nolabel="1"
+                            class="oe_inline"
+                            attrs="{'readonly':[('state','!=','valid')]}"
+                        />
+                    </div>
+                    <notebook>
+                        <page string="Details">
+                            <field name="line_ids" readonly="1">
+                                <tree editable="top">
+                                    <field name="invoice_id" />
+                                    <field name="date_invoice" />
+                                    <field name="taxes_ids" widget="many2many_tags" />
+                                    <field
+                                        name="base_amount"
+                                        string="Document Amount"
+                                    />
+                                    <field name="amount" />
```

Comment:
visualizzare currency_id?

Reply:
Da risolvere in https://github.com/OCA/l10n-italy/issues/2428